pkgs/development/ocaml-modules/uring/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, buildDunePackage
+, ocaml
+, cstruct
+, dune-configurator
+, fmt
+, optint
+, cmdliner
+, logs
+, lwt
+, mdx
+}:
+
+let
+  pname = "uring";
+
+  param = {
+    version = "0.5";
+    sha256 = "sha256-Jc17Myc/oUEYpwUVKlxYSNsAAXn3WuM5AqisFcBG/Xk=";
```

Comment:
`version` need not be wrapped inside `param`; `sha256` is used only once, so might not be needed.